### ISSUE TITLE
No scan  running further - Starting Bruteforce... (3s)

### ISSUE THREAD

**Issue by Archurcl4w** [x90skysn3k/brutespray#133]
Hello, I downloaded the latest release of brutespray on the Kali Linux 6.10.9-amd64 and ran the command as
~~~
brutespray -f nmap.xml -T 10
~~~
brutespray -f nmap.xml

Starting to brute, please make sure to use the right amount of threads(-t) and parallel hosts(-T)...
 ▄ Starting Bruteforce... (3s)
~~~
Even when I increased -T 10 or -T 20, I received the same above message every time I tried. It might be a bug as the  -T  value isn't solving the issue, even after adjusting.

**Comment by x90skysn3k** [comment]
what version are you on?

**Comment by Archurcl4w** [comment]
Thanks, brutespray version is v2.2.4. It will be more helpful if there is a feature to print out a debug output. Something like supported services are not found....quit or skip the scan. Anything else informing the reason or requirements.

**Comment by x90skysn3k** [comment]
Yeah I can work on adding a debug mode. The `-P` flag should print out hosts, ports and services found in the provided file. If you specify a host with `brutespray -H ssh://127.0.0.1` does it run? Supply a known hosts instead of localhost.

**Comment by Archurcl4w** [comment]
Thanks. When it is running with the -P flag, it only prints out "Found Services: IP | Service and Port" in the terminal without any further info. Yeah, it does run, brutespray -H ssh://127.0.0.1 or brutespray -H ssh://ip_for_testing as expected.
